sdk/cosmos/azure-cosmos-kafka-connect/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 #### Bugs Fixed
 
 #### Other Changes
+* Added support to create metadata container if not exists for MasterKey auth - See [PR 39973](https://github.com/Azure/azure-sdk-for-java/pull/39973)
```

Comment:
We should clarify metadata container for which feature (throughput control in this case).

Reply:
the metadata is not related to throughput , the metadata container is used to track metadata (like the containerRids, feedRanges for the container),